### ISSUE TITLE
The implementation of Weight Normalization in Wrapper.py seems wrong

### ISSUE THREAD

**Issue by hwenjun18** [tensorflow/addons#473]
In Wrapper.py, which implements Weight Normalization, but it seems that the code is not complete, such as line 78
'''
        self.g = self.add_variable()

''

and line 60
'''
        self._track_trackable(layer, name='layer')

'''
**and i do not find the implementation of function _track_trackable and add_variable**  , do i miss something?

**Comment by WindQAQ** [comment]
So we inherit from `Wrapper`

The relationship is
```
Wrapper -> Layer -> Module -> AutoTrackable -> Trackable
             ^                                      ^ 
     (add_variable is here)        (_track_trackable is here)
```
`->` means inheritance from.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/wrappers.py#L39
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/engine/base_layer.py#L1705-L1708
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/tracking/base.py#L794-L846

**Comment by seanpmorgan** [comment]
Thanks @hwenjun18! Please re-open this issue if Tzu-Wei's response didn't cover your concern.

**Comment by hwenjun18** [comment]
@seanpmorgan @WindQAQ Thanks for your reply, it covers my concern.
